examples/bots/telegram-bot/Cargo.toml

```diff
@@ -6,7 +6,10 @@ edition = "2021"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-teloxide = { version = "0.12", features = ["macros"] }
+# We need the fix from https://github.com/teloxide/teloxide/pull/1002
+teloxide = { git = "https://github.com/teloxide/teloxide.git", rev = "9c70cc4", features = [
+    "macros",
+] }
```

Comment:
Can you please bump the version of the telegram bot and update the changelog.

Reply:
There is no changelog, should I create one?

Reply:
Yes, create a changelog file.